docs/pages/cloud/architecture.mdx

```diff
@@ -1,40 +1,37 @@
 ---
 title: Teleport Cloud Architecture
-description: Cloud security, availability and networking details.
+description: Cloud security, availability, and networking details.
 ---
 
 ## Security
 
-We have designed the Teleport Cloud environment to be secure; however we are still in the process of
-scrutinizing and executing on our security roadmap and working with independent security
-auditors to identify any gaps. Only once this is complete, the team will evaluate
-whether the Teleport Cloud is ready for strict compliance use-cases.
+We have designed the Teleport Cloud environment to be secure; and, security is an ongoing process of improvement.
+
+Our team routinely scrutinizes, reviews, updates, improves upon, and executes our security roadmap by working with independent security auditors to identify any gaps.
+
+Teleport Cloud only gets the green light for use in strict compliance use-cases only when the team has full confidence that all security gaps or vulnerabilities have been identified and resolved.
 
 ## Compliance
 
-We are going through SOC2 audit that should wrap up by June 2021.
+A SOC2 audit is presently underway and should be concluded by June 2021.
```

Comment:
Can we just update it with the SOC2 audit? @TravisGary

Reply:
SOC2 audit has been completed for the cloud officially. We can state it now.

Reply:
@benarent , @klizhentas - I've synced with @TravisGary - a new blog post has been created (will be published) that we can link to in this section along with a brief snippet.

I believe there's an Issue up to more comprehensively address this in 7.0 - https://github.com/gravitational/teleport/issues/7186 - we'll probably want to update the FAQ, move this section into an include, reuse it in a new section within the Security article. 

I'll hold off on merging this until that's done if that's OK. Should be good here shortly I'd think.